man/io_uring_setup.2

```diff
@@ -106,19 +106,29 @@ call with the following code sequence:
  * and the kernel might not agree on the consistency of the
  * wakeup flag.
  */
-unsigned flags = atomic_load_relaxed(sq_ring->flags);
+unsigned flags = atomic_load_relaxed(ring->sq.kflags);
 if (flags & IORING_SQ_NEED_WAKEUP)
     io_uring_enter(fd, 0, 0, IORING_ENTER_SQ_WAKEUP);
 .EE
 .in
 
 where
-.I sq_ring
+.I ring->sq
 is a submission queue ring setup using the
 .I struct io_sqring_offsets
 described below.
 .TP
 .BR
+Note that, when using liburing, you never directly call the
```

Comment:
From https://unixism.net/loti/tutorial/sq_poll.html

Reply:
This change doesn't make much sense to me - it's just pseudo code. What makes ring->sq more appropriate than an sq_ring variable that points to &ring->sq?

Reply:
ah, I see. I took the example too literally, thanks. 

The spirit of this change is to try and clarify a little which flags field will have `IORING_SQ_NEED_WAKEUP` set. 

Would it make more sense then to just add to the text the flags are in `struct io_uring_sq` `kflags` or there are other possibilities and that would be too restrictive? (I'm new to the world of `io_uring`, mind you)

Reply:
I pushed a new change set, restored the pseudo code. Now that I understand how that part works better, it makes sense as is.